examples/layers/preprocessing/cut_mix_demo.py

```diff
@@ -0,0 +1,46 @@
+"""mix_up_example.py shows how to use the CutMix preprocessing layer to 
+preprocess the oxford_flowers102 dataset.  In this script the flowers 
+are loaded, then are passed through the preprocessing layers.  
+Finally, they are shown using matplotlib.
+"""
+import tensorflow as tf
+from keras_cv.layers.preprocessing.cut_mix import CutMix
+import tensorflow_datasets as tfds
+import matplotlib.pyplot as plt
+
+
+IMG_SIZE = 224
+BATCH = 16
+
+AUTOTUNE = tf.data.AUTOTUNE
+
+
+def resize(image, label):
+    image = tf.cast(image, tf.float32)
+    image = tf.image.resize(image, (IMG_SIZE, IMG_SIZE))
+    return image, label
+
+
+def main():
+    data, ds_info = tfds.load("oxford_flowers102", with_info=True, as_supervised=True)
+    train_ds = data["train"]
+
+    num_classes = ds_info.features["label"].num_classes
+
+    train_ds = (
+        train_ds.map(resize).shuffle(10 * BATCH).batch(BATCH)
```

Comment:
nit, usually the shuffle buffer window should be same/large than the number of images to achieve the perfect shuffle, but it is fine for just a small demo.

Reply:
Yeah let’s just keep it small so that this can run in low memory environments, some reproducibility is nice here too. Maybe I should even remove the shuffle.